terasoluna-gfw-functionaltest-env/pom.xml

```diff
@@ -132,6 +132,7 @@
         <dependency>
             <groupId>org.terasoluna.gfw</groupId>
             <artifactId>terasoluna-gfw-jpa</artifactId>

```

Comment:
Why the dependency on `terasoluna-gfw-jpa` is need in env-project ? 
(This comment is not related to this PR...You do not need to correspond to this.)